config/routes.js

```diff
@@ -90,4 +90,6 @@ module.exports = function(app, passport, auth) {
     app.get('/play', index.play);
     app.get('/', index.render);
 
+    var auth = require('../app/controllers/userAuth');
```

Comment:
### 🔴  Critical 

The variable `auth` has already been used. It has been passed as a parameter in line 3, 
```js
module.exports = function(app, passport, auth) {...}
```
replace this with a unique variable to avoid conflict.